File: leo2-ui/src/main/java/org/deku/leo2/bridge/MessageFactory.java
package org.deku.leo2.bridge;

import org.deku.leo2.rest.entities.internal.v1.Depot;

/**
 * Factory for leobridge messages
 * Created by masc on 21.10.14.
 */
public final class MessageFactory {
    public static Message createViewDepotMessage(Depot depot) {
        Message m = new Message();
        m.put("view", "depot");
        m.put("id", depot.getDepotNr());
        return m;
    }
}

File: leoz-ui/src/main/java/org/deku/leoz/ui/bridge/MessageFactory.kt
package org.deku.leoz.ui.bridge

import org.deku.leoz.service.internal.entity.Station

/**
 * Factory for leobridge messages
 * Created by masc on 21.10.14.
 */
object MessageFactory {
    fun createViewDepotMessage(station: Station): Message {
        val m = Message()
        m.put("view", "depot")
        m.put("id", station.depotNr!!)
        return m
    }
}
